docs/source/guide/tutorials/base/how_to_train/action_classification.rst

```diff
@@ -0,0 +1,275 @@
+        │    │   └── images [105 frames]
+        │    └── Video_2
+        │        ├── annotations.xml
+        │        └── images [64 frames]
+        │
+        └── valid (1530 videos)
+            ├── Video_0
+            │   ├── annotations.xml
+            │   └── images [85 frames]
+            ├── Video_1
+            │   ├── annotations.xml
+            │   └── images [89 frames]
+            └── Video_2
+                ├── annotations.xml
+                └── images [60 frames]
+
+*********
+Training
+*********
+
+1. Choose which action classification model to train by running the following command to see the list of supported templates:
+
+.. note::
+
+  OTX is supporting only X3D model template now, other architecture will be supported in near future.
+
+.. code-block::
+
+  (otx) ...$ otx find --task action_classification
+
+  +-----------------------+----------------------------------+------+----------------------------------------------------------------+
+  |          TASK         |                ID                | NAME |                           BASE PATH                            |
+  +-----------------------+----------------------------------+------+----------------------------------------------------------------+
+  | ACTION_CLASSIFICATION | Custom_Action_Classificaiton_X3D | X3D  | otx/algorithms/action/configs/classification/x3d/template.yaml |
+  +-----------------------+----------------------------------+------+----------------------------------------------------------------+
+
+All commands will be run on the X3D model. It's a light model, that achieves competitive accuracy while keeping the inference fast.
+
+2. Prepare an OTX workspace for the action classification task by running the following command:
+
+.. code-block::
+
+  (otx) ...$ otx build --task action_classification
+  [*] Workspace Path: otx-workspace-ACTION_CLASSIFICATION
+  [*] Load Model Template ID: Custom_Action_Classificaiton_X3D
+  [*] Load Model Name: X3D
+  [*]     - Updated: otx-workspace-ACTION_CLASSIFICATION/model.py
+  [*]     - Updated: otx-workspace-ACTION_CLASSIFICATION/data_pipeline.py
+  [*] Update data configuration file to: otx-workspace-ACTION_CLASSIFICATION/data.yaml
+
+  (otx) ...$ cd ./otx-workspace-ACTION_CLASSIFICATION
+
+It will create **otx-workspace-ACTION_CLASSIFICATION** with all necessery configs for X3D, prepared ``data.yaml`` to simplify CLI commands launch and splitted dataset.
+
+3. Update ``otx-workspace-ACTION_CLASSIFICATION/data.yaml`` file with the path to the CVAT converted HMDB51 dataset.
+
+.. code-block::
+
+  data:
+    train:
+      ann-files: null
+      data-roots: ../data/hmdb51/CVAT/train
+    val:
+      ann-files: null
+      data-roots: ../data/hmdb51/CVAT/valid
+    test:
+      ann-files: null
+      data-roots: null
+    unlabeled:
+      file-list: null
+      data-roots: null
+
+4. To begin training, simply run ``otx train`` from within the workspace directory:
```

Comment:
For the beginner, they must be confused about where the "otx train" should be entered. Therefore, I want the "within the workspace directory" be highlighted(bold)

Reply:
Reflected, thank you for good idea :)